renderer/components/sidebar/index.tsx

```diff
@@ -28,10 +28,10 @@ import {
 import { useToast } from "@/components/ui/use-toast";
 import UpscaylSteps from "./upscayl-tab/upscayl-steps";
 import SettingsTab from "./settings-tab";
-import Footer from "../footer";
+import Footer from "../Footer";
```

Comment:
I didn't notice that the change was wrong 🙈 The old one was fine as is

Reply:
> I didn't notice that the change was wrong 🙈 The old one was fine as is

Hi Nayam.

Sorry did you mean it was fine before this PR?
I was absolutely getting compile errors due to the case sensitive naming.

Reply:
> > I didn't notice that the change was wrong 🙈 The old one was fine as is
> 
> Hi Nayam.
> 
> Sorry did you mean it was fine before this PR? I was absolutely getting compile errors due to the case sensitive naming.

Yeah, it was fine earlier. Footer is the component name and footer is the file name.

Reply:
> Yeah, it was fine earlier. Footer is the component name and footer is the file name.

```
$ git checkout ddd286574b85980ebfea110b275d8f6f339fce69

HEAD is now at ddd2865 Add new model selection
$ npm run dist

> upscayl@2.11.5 dist
> tsc && npm run build && cross-env DEBUG=* electron-builder


> upscayl@2.11.5 build
> tsc && next build renderer

  ▲ Next.js 14.2.10
  - Experiments (use with caution):
    · externalDir

   Linting and checking validity of types  ..Failed to compile.

./components/sidebar/index.tsx:31:20
Type error: Cannot find module '../footer' or its corresponding type declarations.

  29 | import UpscaylSteps from "./upscayl-tab/upscayl-steps";
  30 | import SettingsTab from "./settings-tab";
> 31 | import Footer from "../footer";
     |                    ^
  32 | import { NewsModal } from "../news-modal";
  33 | import Tabs from "../tabs";
  34 | import Header from "../header";
```

I would humbly disagree.